source/libs/on-new-comments.ts

```diff
@@ -7,7 +7,6 @@ const observed = new WeakSet();
 
 const run = debounce(() => {
 	// Safely run all callbacks
-	// eslint-disable-next-line @typescript-eslint/require-await
 	handlers.forEach(async cb => cb());
```

Comment:
```suggestion
	for (const handler of handlers) {
		handler();
	}
```

Reply:
Not all handlers are async. If `handler` errors it will crash RG. Sadly there's no `for async of` to replicate `.foreach(async)`

Reply:
Ok, how about this then? The intent is clearer.

```js
for (const handler of handlers) {
	(async () => {
		try {
			await handler();
		} catch {}
	})();
}
```

Reply:
Sorry Sindre but I really don't see why you insist on making things complicate around async functions. It turns a 5-keyword single line and comment into 10-keyword 7-line that isn't any better to read nor faster to execute. Just why.

Reply:
🤷‍♂ I just prefer explicicity. Feel free to ignore.

Reply:
I like it too, but not in this case. I will extend the comment 😁

Reply:
> I will extend the comment

Instead of letting the code do the explanation...

Reply:
I like explicity too but I don't think it helps in this case.

The semi-equivalent would be this, except the error would technically be considered "caught":

```js
	for (const handler of handlers) {
		try {
			handler();
		} catch (error) {
			console.log(error);
		}
	}
```

But then again I see no advantage over 1-line + comment, which I extended: 
9a1c22b